Kickstarter-iOS/Views/Controllers/ManageViewPledgeRewardReceivedViewController.swift

```diff
@@ -0,0 +1,18 @@
+import Library
+import Prelude
+import UIKit
+
+final class ManageViewPledgeRewardReceivedViewController: ToggleViewController {
```

Comment:
Since we're using `ToggleViewController` as a base class it could be reused in the future and this controller should be able to provide it's own logic (which API call will be called) + styling.